storage/storage_test.go

```diff
@@ -57,7 +58,7 @@ func checkReportForCluster(t *testing.T, storage storage.Storage, orgID types.Or
 }
 
 func writeReportForCluster(t *testing.T, storage storage.Storage, orgID types.OrgID, clusterName types.ClusterName, report types.ClusterReport) {
-	err := storage.WriteReportForCluster(orgID, clusterName, report)
+	err := storage.WriteReportForCluster(orgID, clusterName, report, time.Now())
```

Comment:
not sure whether you want to `time.Now().UTC()`

Reply:
looks like it won't matter because we will have timestamp in db, but I'm not sure too

Reply:
I see, TY